config/webpacker.yml

```diff
@@ -1,7 +1,7 @@
 # Note: You must restart bin/webpack-dev-server for changes to take effect
 
 default: &default
-  source_path: app/frontend
+  source_path: app/javascript
```

Comment:
careful with this, most js and css files are in 'frontend' directory

Reply:
Ok. It was automaticly

Reply:
Revirsed
Addressed here https://github.com/vova/war_shelter/pull/68/commits/4c7d826e300e3dfae4bbdd203fdaf45012028228